### PR TITLE
feat: add `unix_timestamp` udf

### PR DESCRIPTION
**Pull request by zhanghaohit**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

### CODE REVIEW

**Comment by zhanghaohit** [comment]
To get the current timestamp, currently users have to do `unix_timestamp("")`. It's better to support `unix_timestamp()`.

**Review by dl239 — APPROVED** [review]

File: hybridse/src/udf/default_udf_library.cc [hunk 1]
```diff
@@ -2074,6 +2074,40 @@ void DefaultUdfLibrary::InitTypeUdf() {
...
+              - yyyymmdd
+              - yyyy-mm-dd hh:mm:ss
+
+            Example:
+
+            @code{.sql}
+                select unix_timestamp("2020-05-22");
+                -- output 1590076800
+
+                select unix_timestamp("2020-05-22 10:43:40");
```
Comment (zhanghaohit) [inline hybridse/src/udf/default_udf_library.cc:2098]: not support: unix_timestamp("2020-05-22 10:43:40.100") -> float/double

File: hybridse/src/udf/default_udf_library.cc [hunk 2]
```diff
@@ -2074,6 +2074,40 @@ void DefaultUdfLibrary::InitTypeUdf() {
...
+                v1::date_to_unix_timestamp)))
+        .return_by_arg(true)
+        .returns<Nullable<int64_t>>()
+        .doc(R"(
+            @brief Cast date or string expression to unix_timestamp. If empty string or NULL is provided, return current timestamp
+
+            Supported string style:
+              - yyyy-mm-dd
+              - yyyymmdd
+              - yyyy-mm-dd hh:mm:ss
```
Comment (zhanghaohit) [inline hybridse/src/udf/default_udf_library.cc:2090]: another pr to support `format` argument

Comment (zhanghaohit) [reply hybridse/src/udf/default_udf_library.cc:2090]: #2848

**Review by aceforeverd — APPROVED** [review]

**Pull request merged**